.github/workflows/regression-tests.yml

```diff
@@ -100,6 +100,12 @@ jobs:
             python-version: 3.9
             os: ubuntu-20.04
 
+          - sim: icarus
+            sim-version: apt
+            lang: verilog
+            python-version: "3.10"
```

Comment:
Maybe this works?
```suggestion
            python-version: "3.10.0-rc - 3.10"
```

Reply:
Ah yes, range syntax, thanks. I'm going to use https://github.com/actions/setup-python/issues/20#issuecomment-896221379 now as that is proven to work.